lib/shared/drive-constraints.ts

```diff
@@ -95,24 +106,27 @@ export function isDriveLargeEnough(
 		return true;
 	}
 
-	return driveSize >= _.get(image, ['size'], UNKNOWN_SIZE);
+	return driveSize >= (image?.size || UNKNOWN_SIZE);
 }
 
 /**
  * @summary Check if a drive is disabled (i.e. not ready for selection)
  */
 export function isDriveDisabled(drive: DrivelistDrive): boolean {
-	return _.get(drive, ['disabled'], false);
+	return drive?.disabled || false;
```

Comment:
Can the drive be `undefined` ?

Reply:
I don't think so, maybe it was previously so we kept this when transitioning